gradle.properties

```diff
@@ -20,7 +20,7 @@ dockerExecutable=/usr/bin/docker
 apacheCommonsCsvVersion=1.5
 apacheCommonsCompressVersion=1.20
 apacheCuratorVersion=4.0.1
```

Comment:
Wondering if it makes sense to upgrade Apache curator to `4.3.1`, as this version seems quite old. Looking at the release notes of Apache Curator 4.3.0, it seems that ir works with Zookeeper client `3.5.7`: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314425&version=12345141
Maybe it makes sense to upgrade our Zookeeper dependency to `3.5.7` as well (instead of `3.5.9`) to work just with 1 Zookeeper dependency.

Reply:
@RaulGracia -- Should we go ahead with these upgrades in this PR or another PR?